docker/base/run.sh

```diff
@@ -0,0 +1,13 @@
+WORKSPACE=$1
+
+sudo docker run -it --rm \
+    --runtime nvidia \
+    --network host \
+    --privileged \
+    --device /dev/video* \
+    --volume /dev/bus/usb:/dev/bus/usb \
+    --volume /tmp/argus_socket:/tmp/argus_socket \
+    -p 8888:8888 \
+    -v $WORKSPACE:/workspace \
+    --workdir /workspace \
+    jetbot-base:$JETBOT_VERSION
```

Comment:
Do you want to add `--name` option for the base as well? (I see all the other 3 containers are given name)